Packs/ThreatConnect/Integrations/ThreatConnect_v2/ThreatConnect_v2.py

```diff
@@ -695,14 +747,16 @@ def tc_get_indicator_command():
     associated_indicators = json.loads(args['indicator_associations'].lower())
     include_tags = json.loads(args['indicator_tags'].lower())
     include_observations = json.loads(args['indicator_observations'].lower())
+    include_attributes = json.loads(args['indicator_attributes'].lower())
     if 'indicator_type' in args:
         indicator_type = args['indicator_type']
     else:
         indicator_type = None
 
-    ec, indicators, raw_indicators, indicators_associations, indicator_groups, indicator_observations, indicator_tags \
+    ec, indicators, raw_indicators, indicators_associations, indicator_groups, indicator_observations, indicator_tags, indicator_attributes \
         = tc_get_indicator(indicator, owners, rating_threshold, confidence_threshold, associated_groups,
-                           associated_indicators, include_observations, include_tags, indicator_type)
+                           associated_indicators, include_observations, include_tags, indicator_type, include_attributes)
+
```

Comment:
Extend the tc_get_indicator function to allow for requesting indicator attributes